package.json

```diff
@@ -1,25 +1,44 @@
 {
   "name": "sri4node-attachments",
-"version": "2.0.27",
+  "version": "2.0.27",
```

Comment:
This should probably be version 2.1.0 since there are quite a few changes (nothing breaking though, right?).